task/buildpacks-phases/0.3/buildpacks-phases.yaml

```diff
@@ -0,0 +1,486 @@
+      description: Path to the platform directory
+      default: "/platform"
+    - name: CNB_PROCESS_TYPE
+      description: Default process type to set in the exported image
+      default: "web"
+    - name: CNB_RUN_IMAGE
+      description: Reference to an image which is packaging the application runtime to be launched.
+      default: ""
+    - name: CNB_SKIP_LAYERS
+      description: Do not restore SBOM layer from previous image
+      default: false
+    # DEPRECATED: It does not make sense to support such an env variable as mounting the unix docker socket part of a pod from a host volume
+    # will never happen for security reason
+    # - name: CNB_USE_DAEMON
+    #  description: Analyze image from docker daemon
+    #  default: false
+    - name: CNB_USER_ID
+      description: The user ID of the builder image user.
+      default: ""
+
+    - name: APP_IMAGE
+      description: The name of the container image for your application.
+    - name: SOURCE_SUBPATH
+      description: A subpath within the `source` input where the source to build is located.
+      default: ""
+    - name: TAGS
+      description: Additional tag to apply to the exported image
+      default: ""
+    - name: USER_HOME
+      description: Absolute path to the user's home directory.
+      default: /tekton/home
+
+  results:
+    - name: APP_IMAGE_DIGEST
+      description: The digest of the built `APP_IMAGE`.
+
+  stepTemplate:
+    env:
+      - name: CNB_EXPERIMENTAL_MODE
+        value: $(params.CNB_EXPERIMENTAL_MODE)
+      - name: HOME
+        value: $(params.USER_HOME)
+
+  steps:
+    - name: get-labels-and-env
+      image: quay.io/halkyonio/skopeo-jq:0.1.3@sha256:1b3d21ad541227dc9d3e793d18cef9eb00a969c0c01eb09cab88997bc63680c6
+      onError: stopAndFail
+      env:
+        - name: PARAM_VERBOSE
+          value: $(params.CNB_LOG_LEVEL)
+        - name: PARAM_BUILDER_IMAGE
+          value: $(params.CNB_BUILDER_IMAGE)
+        - name: PARAM_CNB_PLATFORM_API
+          value: $(params.CNB_PLATFORM_API)
+        - name: PARAM_CNB_PLATFORM_API_SUPPORTED
+          value: $(params.CNB_PLATFORM_API_SUPPORTED)
+      results:
+        - name: UID
+          description: UID of the user specified in the Builder image
+        - name: GID
+          description: GID of the user specified in the Builder image
+        - name: EXTENSION_LABELS
+          description: "Extensions labels: io.buildpacks.extension.layers defined in the Builder image"
+        - name: CNB_PLATFORM_API
+          description: The CNB_PLATFORM_API to be used by lifecycle and verified against the one supported by this task
+      script: |
+        #!/usr/bin/env bash
+        set -eu
+
+        if [ "${PARAM_VERBOSE}" = "debug" ] ; then
+          set -x
+        fi
+
+        echo # Check if registry creds docker file has been mounted from a secret"
+        if [[ -f "$HOME/.docker/config.json" ]]; then
+          printf %"s\n" "The docker config.json file exists !"
```

Comment:
apologies for my ignorance but I am unable to find the location from where we are mounting the credentials to this task. Can you please point me to the right location? 😅

Reply:
again apologies, I was able to find the details in README 😅